visualizer-server/config.toml

```diff
@@ -0,0 +1,6 @@
+[server.grpc]
```

Comment:
what's the idea difference between this and config/base.toml?

Reply:
oops, forgot to add this file to gitignore